doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keystore-default-con.adoc

```diff
@@ -1,13 +1,13 @@
 [id='managing-business-central-ssh-keystore-default-con']
 = Default SSH keystore
 
-The default SSH keystore included with {CENTRAL} provides a file-based storage mechanism to store a user's public keys. By default, {CENTRAL} uses the `*.security` folder as the root directory. However, it is possible to use a custom storage path by setting the value of the `appformer.ssh.keys.storage.folder` system property point to a different folder.
+The default SSH keystore included with {CENTRAL} provides a file-based storage mechanism to store a user's public keys. By default, {CENTRAL} uses the `*.security` folder as the root directory. However, it is possible to use a custom storage path by setting the value of the `appformer.ssh.keys.storage.folder` system property to point to a different folder.
```

Comment:
However, you can also use a custom storage path....

Reply:
Done